orttraining/orttraining/python/training/execution_agent.py

```diff
@@ -77,9 +77,17 @@ class TrainingAgent(object):
     This is the main class used to run an ORTModule model training.
     """
 
-    def __init__(self, path_or_bytes, session_options=None, providers=None, provider_options=None):
+    def __init__(self, path_or_bytes, fw_feed_names, fw_fetches_names, fw_outputs_device_info,
+                 bw_feed_names, bw_fetches_names, bw_outputs_device_info, session_options=None,
+                 providers=None, provider_options=None):
         """
         :param path_or_bytes: filename or serialized ONNX or ORT format model in a byte string
+        :param fw_feed_names: Feed names for foward pass.
```

Comment:
ORTModule actually uses google style docstring. If you could follow those convetions, would make the auto documentation generation a lot easier. We are going to publish them to the public soon

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Not that this class is not part of the public API, so we should not be generating public documentation for it.

(In general, I'm in favor of using consistent style across the codebase, but it looks like Zeeshan followed the local style for this class, and we should clean it up as a separate pass.)